src/main/java/by/exadel/internship/team_four/entity/InternshipLocation.java

```diff
@@ -0,0 +1,17 @@
+package by.exadel.internship.team_four.entity;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class InternshipLocation {
```

Comment:
@viktarkolbik what do you think?
```suggestion
public class Location {
```

Reply:
Form has fields country and city, and we used this name to understand this location for internship

Reply:
Okay, let's leave as it is now. In future if this class can be reused in other dto we can make generic this class to LocationReference and share among other dtos.

Reply:
done